pkg/client/images.go

```diff
@@ -61,3 +62,10 @@ func (c *images) Delete(name string) (err error) {
 	err = c.r.Delete().Resource("images").Name(name).Do().Error()
 	return
 }
+
+// UpdateSignatures allows to modify signatures of given image.
+func (c *images) UpdateSignatures(image *imageapi.Image) (result *imageapi.Image, err error) {
```

Comment:
Seems like this is more `CreateSignatures`.  It's updating an underlying object, but you're actually creating a thing, right?